TerrorConsole/Assets/_Main/Scripts/Objects/Switchs/PuzzleSwitchLever.cs

```diff
@@ -0,0 +1,37 @@
+using UnityEngine;
+
+namespace TerrorConsole
+{
+    public class PuzzleSwitchLever : SwitchObject
```

Comment:
This is basically a `SwitchLever`, reuse the code, this script should be erased.